fe/src/components/Tab/HostTab.vue

```diff
@@ -1,47 +1,53 @@
 <template>
-  <v-app class="pl-10">
+  <v-app>
     <v-navigation-drawer permanent>
       <v-list dense>
         <v-list-item-group>
-
           <!--Listings-->
-          <v-list-item href="#listings" @click="changeActive('list')" :class="{'option-active' : isActive}">
+          <v-list-item
+            href="#listings"
+            @click="changeActive('list')"
+            :class="{ 'option-active': isActive }"
+          >
             <v-list-item-content>
               <v-list-item-title><b>Your Listings</b></v-list-item-title>
             </v-list-item-content>
           </v-list-item>
 
           <!--Reservations-->
-          <v-list-item href="#reservations" @click="changeActive('res')" :class="{'option-active' : !isActive}">
+          <v-list-item
+            href="#reservations"
+            @click="changeActive('res')"
+            :class="{ 'option-active': !isActive }"
+          >
             <v-list-item-content>
               <v-list-item-title><b>Your Reservations</b></v-list-item-title>
             </v-list-item-content>
           </v-list-item>
-
         </v-list-item-group>
       </v-list>
     </v-navigation-drawer>
   </v-app>
 </template>
 
 <script setup>
-  let isActive = true
-
-  function changeActive(option) {
+let isActive = true;
 
-    switch(option){
-      case "list": isActive = true; 
+function changeActive(option) {
+  switch (option) {
```

Comment:
is parameter `option` only has two type? if yes, please change to boolean and you can change name to `isListing` or `isReservation`

Reply:
Changed it already maam